lib/rust/parser/debug/src/lib.rs

```diff
@@ -122,10 +122,18 @@ fn strip_hidden_fields(tree: Value) -> Value {
         ":spanLeftOffsetVisible",
         ":spanLeftOffsetCodeReprBegin",
         ":spanLeftOffsetCodeReprLen",
-        ":spanLeftOffsetCodeUtf16",
+        ":spanLeftOffsetCodeLenUtf8",
+        ":spanLeftOffsetCodeLenUtf16",
+        ":spanLeftOffsetCodeLenNewlines",
+        ":spanLeftOffsetCodeLenLineChars16",
```

Comment:
If the node has multiple lines, this is the length of the last line?

Reply:
Yes, a line:col length contains the information necessary to reach an end line:col location from a start location.